diff --git a/templates/pro_hun/train.py b/templates/pro_hun/train.py
index e62ec95..0304567 100644
--- a/templates/pro_hun/train.py
+++ b/templates/pro_hun/train.py
@@ -294,12 +294,10 @@ if __name__ == "__main__":
                 # if early_ind == 2:
                 #     flag = False
                 #     break
-        break
         torch.save(mnist_resnet, os.path.join(dirname, f"model_mnist{i}.pickle"))
         print("학습 종료!")
         print(f"최고 accuracy : {best_test_accuracy}, 최고 낮은 loss : {best_test_loss}")
     ed_time = time.time()
-    # total_minute = (round(ed_time - st_time, 2)) // 60
     total_minute = (round(ed_time - st_time, 2)) // 60
     print(f"총 학습 시간 : {total_minute}분 소요되었습니다.")
     notification(best_test_accuracy)
diff --git a/templates/wandb/latest-run b/templates/wandb/latest-run
index de08289..69a7e70 120000
--- a/templates/wandb/latest-run
+++ b/templates/wandb/latest-run
@@ -1 +1 @@
-run-20210901_044542-25ac32g2
\ No newline at end of file
+run-20210901_050642-20vtfzgm
\ No newline at end of file
